### PR TITLE
Sarah afiya/issue 12/determine variable types

### PR DESCRIPTION
**Pull request by sarah-afiya**
issue #12

### CODE REVIEW

**Comment by snadi** [comment]
In the future, it would be good to include a summary of the changes in the PR so it's clear how you addressed the issue (like on a high level, what is the implemented solution)

File: src/analyze/TaskExtractor/extractor.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def get_paragraphs_and_tasks(paragraphs, task_file, property_file, domain_filter
 
 
 def preprocess(text):
-    code = re.compile(r"(?:<code|<pre).*?>(.*?)</(?:code|pre)>")
+    code = re.compile(r"(?:<src|<pre).*?>(.*?)</(?:src|pre)>")
```
[the remark this answers is not in the document]
Comment (sarah-afiya) [reply src/analyze/TaskExtractor/extractor.py:91]: Sorry, I think this change was due to renaming the directory. I'll revert the commit

File: src/analyze/method_linker/python_matcher.py [hunk 2]
```diff
@@ -277,3 +283,46 @@ def python_match_examples(repo_name, examples, functions, classes):
                 seen.add((link[0], link[1]))
 
     return method_calls
+
```
[the remark this answers is not in the document]
Comment (sarah-afiya) [reply src/analyze/method_linker/python_matcher.py:286]: yes

File: Summary/analyze/MethodLinker/python_matcher.py [hunk 3]
```diff
@@ -277,3 +282,46 @@ def python_match_examples(repo_name, examples, functions, classes):
                 seen.add((link[0], link[1]))
 
     return method_calls
+
+
+def get_declared_variable_mapping(example_code, classes):
```
Comment (snadi) [inline Summary/analyze/MethodLinker/python_matcher.py:287]: maybe I missed it but where is this function called/used?

Comment (sarah-afiya) [reply Summary/analyze/MethodLinker/python_matcher.py:287]: It's not used anywhere in this PR. It is used in the next PR #15

**Review by snadi — APPROVED** [review]

**Pull request merged**